asset-registration/model_and_dataset_registration.py

```diff
@@ -0,0 +1,34 @@
+from azure.ai.ml import MLClient
+from azure.identity import DefaultAzureCredential
+
+import json
+
+if __name__ == '__main__':
+    with open("./config.json") as f:
+        config = json.load(f)
+
+    credential = DefaultAzureCredential()
+
+    # Get a handle to the workspace
+    ml_client = MLClient.from_config(
+        credential=credential,
+    )
+    
+    # Get a handle to the registry
+    ml_client_registry = MLClient(credential=credential,
+                        registry_name=config["registry_name"],
+                        registry_location=config["registry_location"])
+
+    # Share model. make sure the models have the same version in the local and global registries
+    try:
+        ml_client.models.share(name=config["model_name"], version=config["model_version"], registry_name=config["registry_name"], share_with_name=config["model_name"], share_with_version=config["model_version"])
+    except Exception as e:
+        print(e)
```

Comment:
Why print the exception?
Are we saying that all exceptions are fine, and regardless of the exception we'd continue processing.
If the script should stop at this point, I suggest you remove the try/except and allow the script to naturally print the exception as part of the stack track.